<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Serial</class>
 <widget class="QMainWindow" name="Serial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="gbHard">
      <property name="title">
       <string>Configuração de hardwere</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QToolButton" name="tbRefrech">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="lbPort">
             <property name="text">
              <string>Port</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbPort"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="lbBaud">
             <property name="text">
              <string>Baud rate</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbBaud"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>54</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="lbDataBit">
             <property name="text">
              <string>Data bits</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbDataBit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="lbPari">
             <property name="text">
              <string>Paridade</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbPari"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>53</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="lbStop">
             <property name="text">
              <string>Stop bit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbStop"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="lbFlow">
             <property name="text">
              <string>Controle de fluxo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cbFlow"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPlainTextEdit" name="pteMonitor">
      <property name="styleSheet">
       <string notr="true">QPlainTextEdit {
    background-color: #121212;      /* Fundo quase preto (estilo Dark Mode) */
    color: #00FF00;                 /* Texto verde &quot;Matrix/Terminal&quot; para destaque */
    border: 2px solid #333333;      /* Borda discreta */
    border-radius: 4px;
    font-family: 'Consolas', 'Monospace', 'Courier New'; /* Fontes monoespaçadas */
    font-size: 10pt;
    padding: 5px;
}

/* Estilizando a barra de rolagem para combinar */
QScrollBar:vertical {
    border: none;
    background: #222222;
    width: 10px;
}
QScrollBar::handle:vertical {
    background: #444444;
    min-height: 20px;
}</string>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::LineWrapMode::NoWrap</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="centerOnScroll">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #3498db;
border: 2px solid gray;
border-radius: 10px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbStatus">
        <property name="styleSheet">
         <string notr="true">/* No Label de Status */
QLabel {
    color: #AAAAAA;
    font-weight: bold;
    font-size: 11px;
    padding-left: 10px;
}
</string>
        </property>
        <property name="text">
         <string>status</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
